### ISSUE TITLE
Thunderbird Layan theme

### ISSUE THREAD

**Issue by alexanderadam** [vinceliuice/Layan-gtk-theme#46]
I just thought whether you would be able to achieve such amazing transformation with Thunderbird as well?

**Comment by BirsanStefan** [comment]
Oh, I second this. This is an awesome theme, I would love to see it on Thunderbird.

**Comment by Aman9das** [comment]
I don't think this would be implemented by Vince, he's a designer and knowledgeable about gtk and qt theming. Porting his themes to individual apps would be inefficient use of his expertise, I think these could be a community project of sorts bringing the themes to life in other applications
